### PR TITLE
History Background Work

### PR DESCRIPTION
**Pull request by kuehma02**
This changes many of the current api calls to allow for building of a history page

### CODE REVIEW

File: server/services/goal-service.js [hunk 1]
```diff
@@ -16,11 +15,31 @@ function get(req, res) {
     })
 }
 
+// function getOnMYid (req, res) {
+//   console.log(req);
+//   const {userId, mmyyId} = req.params;
```
Comment (rublan01) [inline server/services/goal-service.js]: Maybe let's look into deleting this if applicable?

File: server/services/test [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+// function getOnMYid (req, res) {
+//   console.log(req);
+//   const {userId, mmyyId} = req.params;
+//   const docs = Goal.find({userId: userId, monthYearId: mmyyId}).read(ReadPreference.NEAREST);
+//   return docs
+//     .then(goals => {
+//       res.json(goals);
+//     })
+//     .catch(err => {
+//       res.status(500).send(err);
```
Comment (rublan01) [inline server/services/test]: Also this

File: server/services/goal-service.js (hunk 1 above)
Comment (kuehma02) [reply server/services/goal-service.js]: Done!

File: server/services/test (hunk 2 above)
Comment (kuehma02) [reply server/services/test]: Done!

**Pull request merged**